nix/opam2nix.nix

```diff
@@ -0,0 +1,4 @@
+import (builtins.fetchGit {
+  url = "https://github.com/timberton/opam2nix.git";
```

Comment:
Looks like this should say "timbertson", not "timberton".

Reply:
Fixed. Thanks.